app/views/hiring_staff/schools/show.html.haml

```diff
@@ -57,5 +57,5 @@
           %td
             %strong.strong Website
           %td
-            = link_to @school.url, @school.url
+            = link_to "#{@school.name} website", @school.url
```

Comment:
Just following the DAC recommendation…

<img width="522" alt="screen shot 2018-06-06 at 14 57 34" src="https://user-images.githubusercontent.com/822507/41042601-002deffe-699a-11e8-9b34-1fc5a56f08d6.png">

Reply:
Given the description of this flagging screen readers not being unable to understand the text, `aria-label` seems to be a better option as it doesnt impact existing behavior - this page has already been verified by hiring schools with no such issue flagged. 

Also, the solution seems to be a generic suggestion that would work for links within text but in this instance there is already the context of website.

Also we should also use `target=_blank` to open external websites, and a better text such as `Opens #{school.name} website in new window`

Reply:
I don't think showing the full URL in this case is optimal; I'm thinking specifically about edge cases with very long URLs, which may break onto more than one line and potentially be confusing for visually impaired users. 

On the edit and review pages there is a case for showing the full URL so it can be checked at a glance by hiring staff.

I do agree that your example text is better, though.